README.md

```diff
@@ -46,7 +68,11 @@ var Panda = function(name, age) {
 ```
 Your Answer:
 ```js
-// your code here
+Panda.prototype = {
+  eat_bamboo: function() {
+    this.num_bamboo_eaten++;
```

Comment:
Yes! just be careful when defining a prototype method in the object literal notation as above. If there were previously defined methods they would be overwritten using this syntax. I would caution just to use this syntax instead to be safe: 
```
Panda.prototype.eat_bamboo = function(){
  this.num_bamboo_eaten += 1;
}
```